test/framework/deployments/tracing/kubernetes.go

```diff
@@ -17,7 +17,7 @@ type k8SDeployment struct {
 var _ Deployment = &k8SDeployment{}
 
 func (t *k8SDeployment) ZipkinCollectorURL() string {
-	return fmt.Sprintf("http://jaeger-collector.%s:9411/api/v2/spans", framework.Config.DefaultObservabilityNamespace)
+	return fmt.Sprintf("http://jaeger-collector.%s:9411/api/v2/spans", framework.Config.DefaultTracingNamespace)
```

Comment:
Won't this break as we're using the new tool and the tracing gets installed in the obs namespace?

Reply:
forgot to revert that while copying stuff, good catch, thanks!

Reply:
Yes, that did break the tests, so they were not flaky but actually catching this regression! Fix on the way.